C-Sharp/search/Fibonacci_Search.cs

```diff
@@ -0,0 +1,94 @@
+// C# program for Fibonacci Search 
+// Time Complexity - O(log(n))
+// Space Complexity - O(1)
```

Comment:
- Remove the line
- Add Multi-Line Comments 
- Add a beginner-friendly description at the top